### ISSUE TITLE
Excel Export - prependContent with height on ExcelRow is being ignored

### ISSUE THREAD

**Issue by vsdotnetguy** [ag-grid/ag-grid#5594]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of AG Grid**

React Enterprise Trial on behalf of paying client

**Providing a Reproducible Scenario**

Configure Excel export to use prependContent param and pass it an array of ExcelRow with height value set.  In v28 the value is being ignored as far as being applied within the Excel file.

If reporting a bug make sure to state.

Current behavior.
Expected behavior. If possible back this up with our docs/examples if possible

**Current behavior**
setting the height value within the ExcelRow object seems to be ignored when prepending content

**Expected behavior**
setting the height value within the ExcelRow object is applied and the row height is correctly set when the Excel file is opened.

**Please tell us about your environment:**
Windows 10, Visual Studio 2022, NPM, IIS Express

* **AG Grid version:** 28.1.0

* **Browser:**
Latest Chrome

* **Language:** [jsx]

**Comment by StephenCooper** [comment]
Hi,

Do you think you could reproduce the issue for us based off one of the examples on this page? https://ag-grid.com/react-data-grid/excel-export-extra-content/

That would help us investigate the issue.

**Comment by vsdotnetguy** [comment]
Hi Stephen,

I was able to reproduce this simply by adding height: 50 property to any or all of the getRows ExcelRow instances in your referenced example and it does not work when running in Plunker.

Thanks...Bob Baldwin (aka VSDotNetGuy)

**Comment by StephenCooper** [comment]
Ok so would appear this reproduces the issue as the heights do not make a difference. https://plnkr.co/edit/gbWfQ5QYgQFUqq0i?open=index.tsx

**Comment by StephenCooper** [comment]
I have raised this in our support system now that we can reproduce the issue. Thanks for reporting!

**Comment by StephenCooper** [comment]
Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference: AG-7316

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

See whether this item will be part of our next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-7316) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/

Thanks again for bringing this up with us - we appreciate your time!
